autosklearn/automl.py

```diff
@@ -221,6 +240,27 @@ def __init__(self,
         # After assigning and checking variables...
         # self._backend = Backend(self._output_dir, self._tmp_dir)
 
+    def _create_dask_client(self):
+        self._is_dask_client_internally_created = True
+        processes = False
```

Comment:
The only problem is if you requested 1 job. If you create a client with 1 job and process true, in my laptop, the client creation goes crazy and forces me to terminate python immediately. But also, I think this will allow us to run on a no multiple cores for debug purposes.

This can be changed to force jobs to be always more than 2 and process equal true always. Let me know your preferences.

Reply:
I like the changes, they make sense.